tests/integ/modin/resample/test_resample.py

```diff
@@ -65,6 +65,26 @@ def test_resample_date_before_snowflake_alignment_date():
     )
 
 
+@sql_count_checker(query_count=2, join_count=1)
+def test_resample_date_general_ntz():
```

Comment:
@sfc-gh-lmukhopadhyay We already have this test `test_resample_date_wraparound_snowflake_alignment_date` so I think it's okay not to have this

Reply:
That's true I'll remove this